northstar/hello_world/Cargo.toml

```diff
@@ -0,0 +1,7 @@
+[package]
+name = "hello_world"
+version = "0.1.0"
+authors = ["ESRLabs"]
+edition = "2018"
+
+[workspace]
```

Comment:
We should get rid of one `hello`. There's examples/containers/hello` which is the very same as this here.